protocol/protobuf/pairing.proto

```diff
@@ -223,9 +223,11 @@ message SyncSavedAddress {
   bytes  address = 1;
   string name = 2;
   bool   favourite = 3;
-  uint64 chain_id = 4;
   bool   removed = 5;
   uint64 update_clock = 7;
```

Comment:
@IvanBelyakoff maybe we could use just `clock`, just for the sake of consistency, to stick to the way how it's done for other messages.

Reply:
If that's the request than ok. Will we have to select among sort criteria, like `newest first`/`most recently updated first`? If there are both options, then ok, but if there is no options I guess it should be sort by `most recently updated first`.

Reply:
If I remember correctly, for other implementations I had as an example, the clock is coming from the messenger (lamport clock maybe?). In the case of the saved address, this is the user's wall clock from when the entry was updated independent of sync or messenger. It is used to solve editing conflicts.